Jenkinsfile

```
@@ -1082,11 +1082,13 @@ pipeline {
                             rpm_test_post(env.STAGE_NAME, env.NODELIST)
                         }
                     }
-                } // stage('Test CentOS 7 RPMs')
+                } // stage('Test RPMs on EL 8.6')
                 stage('Test RPMs on Leap 15.4') {
                     when {
                         beforeAgent true
-                        expression { ! skipStage() }
+                        // Workaround for SRE-1993 - skip this stage if a specific RPM version is specified
+                        // expression { !skipStage() }
+                        expression { params.CI_RPM_TEST_VERSION == '' }
```

Here rather than `skipStage()` because `release/2.4` is the only victim of this?  Wouldn't this happen on `master` also for the case of using `CI_RPM_TEST_VERSION` or `RPM-test-version:`?  But speaking of which:

```suggestion
                        expression { ! skipStage{} && rpmTestVersion == '' }
```

---

Since we call skipStage() w/o arguments we fall into https://github.com/daos-stack/pipeline-lib/blob/master/vars/skipStage.groovy#L167 where there is no entry that matches `'Test RPMs on Leap 15.4'`, so we always land on the default https://github.com/daos-stack/pipeline-lib/blob/master/vars/skipStage.groovy#L552.

In other words `! skipStage()` for  `'Test RPMs on Leap 15.4'` is always True.

---

Yes, you would of course need to add an entry for `Test Leap 15.4 RPMs` just as `Test Leap 15.2 RPMs` was added when that stage was being added to DAOS pipelines.